config.example.json

```diff
@@ -0,0 +1,25 @@
+{
+  "videoSettings": {
+    "allowBackgroundEffects": true,
+    "allowCameraControl": true,
+    "allowVideoOnJoin": true,
+    "defaultResolution": "1280x720"
+  },
+  "audioSettings": {
+    "allowAdvancedNoiseSuppression": true,
+    "allowAudioOnJoin": true,
+    "allowMicrophoneControl": true
+  },
+  "waitingRoomSettings": {
+    "allowDeviceSelection": true
+  },
+  "meetingRoomSettings": {
+    "defaultLayoutMode": "active-speaker",
+    "showArchiveButton": true,
+    "showCaptionsButton": true,
+    "showChatButton": true,
+    "showEmojiButton": true,
+    "showParticipantList": true,
```

Comment:
FYI, we omit `Button` from this config value because we don't want to show the participant list from the hidden participants tile either